src/hooks/useResolveWeb3Name.ts

```diff
@@ -13,21 +14,23 @@ import { AppContextType } from 'types/context';
 import { appConfig } from '../config/index.js';
 import { getUdResolver } from 'helpers/w2w/udResolver';
 
-const getEnsName = async (
-  provider: ethers.providers.BaseProvider | any,
-  checksumWallet: string,
-  setWeb3NameList: any
-) => {
-  let ensName: string = '';
-  provider.lookupAddress(checksumWallet).then(async (ens) => {
-    if (ens) {
-      ensName = ens;
-      setWeb3NameList((prev) => ({ ...prev, [checksumWallet]: ens }));
-    } else {
-      ensName = null;
-    }
-  });
-  return ensName;
+const getDomainName = async (checksumWallet: string, setWeb3NameList: any) => {
```

Comment:
If we have this functionality inside the sdk library then why not use it from there and avoid installing the space id library here?

Reply:
not sure why its not getting installed from the sdk itself, even though its mentioned in dependencies in uiweb package